components/pkg-export-kubernetes/src/main.rs

```diff
@@ -221,6 +221,7 @@ fn cli<'a, 'b>() -> App<'a, 'b> {
         .add_builder_args()
         .add_tagging_args()
         .add_publishing_args()
+        .add_pkg_ident_arg(false)
```

Comment:
It's not clear what `false` does here. Either an `Enum` as @krnowak suggested or a `struct` would be an improvement I think.